extensions/table/table_entry.go

```diff
@@ -33,15 +37,9 @@ func (t TableEntry) generateIt(itBody reflect.Value) {
 		}
 	}
 
-	body := func() {
+	ginkgo.ExplicitItNode(t.Description, func() {
 		itBody.Call(values)
-	}
-
-	if t.Focused {
-		ginkgo.FIt(t.Description, body)
-	} else {
-		ginkgo.It(t.Description, body)
-	}
+	}, t.Flag, t.codeLocation, 0)
```

Comment:
The last `0` argument does not look correct?
You pass it further to `ExplicitItNode` where it becomes the timeout for `PushItNode` which would result in a timeout of 0 seconds [here](https://github.com/onsi/ginkgo/blob/1ea49cf36e84a951dd339c6da927162766ac81c4/ginkgo_dsl.go#L622) rather than the default timeout of 1 second defined [here](https://github.com/onsi/ginkgo/blob/1ea49cf36e84a951dd339c6da927162766ac81c4/ginkgo_dsl.go#L620).